mappings/net/minecraft/util/ProgressListenerImpl.mapping

```diff
@@ -0,0 +1,8 @@
+CLASS mk net/minecraft/util/ProgressListenerImpl
+	FIELD b instance Lnet/minecraft/client/Minecraft;
```

Comment:
```suggestion
	FIELD b minecraft Lnet/minecraft/client/Minecraft;
```

Reply:
why

Reply:
don't we call minecraft minecraft

Reply:
I always named these things instance. @Blayyke wdy think

Reply:
we use minecraft everywhere else i think

Reply:
what should the convention be

Reply:
minecraft plz
or we will have a ton of `instance` in guis that are not gui instances

Reply:
i prefer minecraft because instance is used for public static final singleton fields

Reply:
ok